### DISCUSSION TITLE
prblem use with xp pen

### DISCUSSION THREAD

**Discussion by natank07** [opentoonz/opentoonz#6737] [General]
Hy when i use opentoonz one my scream it ok, but when i want to use on my new tablette xp pen 13.3 pro, it not working, i can do anythink with the pen (open files, change colors etc...) all this ok, it look like working ok, but when i want to draw it s not working, when i start something, it comes the draws i did 10 or 20s after and not at the same place. Same with the zoom boton, it s working if the program is on my screm, but not on the scream ok the tablette...
I try with other drawing program it s ok, working normal...
I m very sad because i bought this tablette to use opentools but after 1 week trying..
I use windows 10, ryzen 5 2600, gtx 1080 etc..
sorry for my english
Nathan

**Comment by manongjohn** [comment]
When drawing with the pen on the tablet, is it moving the screen instead? If yes, Enable Windows Ink Support in OT's Preferences -> Tablet/Touch Settings, then close and restart OpenToonz.

**Comment by Gumball-Waterson** [comment]
Hi, I'm having the exact same problem with my tablet too (also an xp-pen 13.3 pro). But only on my macbook, I used to use a windows computer before hand and had no problem with it. But only now with the new computer is this coming up.  I try to draw something, nothing happens for about 10-20 seconds, then it appears in a different place than where I originally drew it.  Although other drawing programs and 3d programs such as blender seem to be working fine as usual. if anybody new what was going on that would be greatly appreciated. 

Quatour Temporum

**Comment by Gumball-Waterson** [comment]
I tried downloading an older version of opentoonz (v1.3) to see if maybe it was a bug with the new v1.4, cause that used to work for me just fine with windows. But I've run into a weird problem where if I use raster levels it gives me the same problem I had before, where as if I use vector levels, there's a delay of about 5 seconds or so, but it's in the same spot as where I drew it instead of being off somewhere else.

Quatour Temportum

**Comment by DavisVanLuven** [comment]
Link to drive video of issue:
https://drive.google.com/file/d/1IiiSXJ0NbX3eBYjX-laZcJCCUD9QPcqI/view?usp=sharing

system:
Macbook Pro 13 inch 2017
Big Sur 11.2.3
Intel Iris Plus Graphics 640 1536 MB
Display:Resolution:Default for Display

Tablet:
XP-Pen Artist 15.6 Pro
Up-to-date drivers

Description:
- When the interface is on my computer screen, I can draw with the trackpad without issue
- When I move the interface to the tablet, the display is enlarged and the pen marks I made are pushed off to the side and pixilated
- When zooming in and out (on tablet) the pen markings move independent from the canvas 
- When drawing (on tablet) the marks will appear after a moment but way off to the upper right corner and they will be skewed, enlarges, and pixilated
- If I drag the UI back to the Mac screen, everything is fine again

I would be very excited to use this program. It's the best open source animation program I could find for classic 2d cel. Please let me know if you have any fixes or updates. Thank you

**Comment by Francoislittle** [comment]
Bonjour, ma Tablette aussi fait des siennes, une Veikk A50. OTooz la reconnait mais je ne peux faire que des points, pas des lignes !!

**Comment by orlyk2d** [comment]
change your graphic display to Main display
